core/sql/cli/memorymonitor.cpp

```diff
@@ -102,12 +102,18 @@ MemoryMonitor::MemoryMonitor(Lng32 windowSize,
 {
   // if the windowSize is 0, we do not need memory monitor.
   assert(windowSize);
-  char buffer[1024];
+  char buffer[2048];
   char *currPtr;
   size_t bytesRead;
   fd_meminfo_ = fopen("/proc/meminfo", "r");
   if (fd_meminfo_) {
-    bytesRead = fread(buffer, 1, 1024, fd_meminfo_);
+    bytesRead = fread(buffer, 1, 2048, fd_meminfo_);
```

Comment:
Can I suggest to use sizeof(buffer) here instead of 2048? Same for lines 230 and 271 below.